Failed to build on Linux (Ubuntu 20.04, clang-12): `fatal error: 'span' file not found`

## Summary

I'm getting a build failure on Linux regarding `'span' file not found`.

I'll try building with GCC instead but haven't yet.


## Versions

+ Cemu git revision: f3ff919
+ Linux:
    ```console
    $ lsb_release -a
    No LSB modules are available.
    Distributor ID:	Ubuntu
    Description:	Ubuntu 20.04.5 LTS
    Release:	20.04
    Codename:	focal
    ```
+ clang-12:
    ```console
    $ clang-12 --version
    Ubuntu clang version 12.0.0-3ubuntu1~20.04.5
    Target: x86_64-pc-linux-gnu
    Thread model: posix
    InstalledDir: /usr/bin
    ```
+ cmake:
    ```console
    $ cmake --version
    cmake version 3.25.2
    
    CMake suite maintained and supported by Kitware (kitware.com/cmake).
    ```


## Steps to Reproduce / Summary of Steps Taken

I followed the "Build from Source" docs.

1. Clone Cemu: `git clone --recursive https:// ...`
2. `cd Cemu`
3. Installl prereqs: `sudo apt install cmake curl freeglut3-dev git libgcrypt20-dev libgtk-3-dev libpulse-dev libsecret-1-dev libsystemd-dev nasm ninja-build`
4. Run `cmake` command
5. Get an error about `clang`
6. Install `clang-12`
7. Run `cmake` again
8. Get the `CMAKE_BUILD_WITH_INSTALL_RPATH` error
9. Re-run `cmake` and include the `RPATH` cli arg
10. Success!
11. Run `cmake --build build`

## Expected result

A successful build.

## Actual Result

Some warnings about comparing integers `int` vs `unsigned int` - who cares.

Errors building the following files:
+ [89/491] Building CXX object src/CMakeFiles/CemuBin.dir/cmake_pch.hxx.pch
+ [90/491] Building CXX object src/input/CMakeFiles/CemuInput.dir/cmake_pch.hxx.pch
+ [91/491] Building CXX object src/Cafe/CMakeFiles/CemuCafe.dir/cmake_pch.hxx.pch
+ [92/491] Building CXX object src/gui/CMakeFiles/CemuGui.dir/cmake_pch.hxx.pch
+ [93/491] Building CXX object src/Common/CMakeFiles/CemuCommon.dir/cmake_pch.hxx.pch
+ [94/491] Building CXX object src/Cemu/CMakeFiles/CemuComponents.dir/cmake_pch.hxx.pch

Each file had the same error:

```
/home/dougthor42/dev/Cemu/src/Common/precompiled.h:76:10: fatal error: 'span' file not found
#include <span>
         ^~~~~~
1 error generated.
ninja: build stopped: subcommand failed.
```


## Full Logs

See next post - went over the character limit :laughing:

Looks like that did the trick! I'll submit a PR to update the build docs tomorrow. Thanks!

I went with `sudo apt install libstdc++-10-dev`. There were many other options ranging from libstdc++5 to libstdc++6-10 to libstdc++9

<details>
<summary>cmake --build build</summary>

```console
$ cmake --build build
[187/403] Building CXX object src/Cafe/CMakeFiles/CemuCafe.dir/IOSU/legacy/iosu_crypto.cpp.o
/home/dougthor42/dev/Cemu/src/Cafe/IOSU/legacy/iosu_crypto.cpp:151:22: warning: 'EC_KEY_new_by_curve_name' is deprecated [-Wdeprecated-declarations]
        EC_KEY* ec_pubKey = EC_KEY_new_by_curve_name(NID_sect233r1);
                            ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:998:1: note: 'EC_KEY_new_by_curve_name' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 EC_KEY *EC_KEY_new_by_curve_name(int nid);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cafe/IOSU/legacy/iosu_crypto.cpp:152:10: warning: 'EC_KEY_set_public_key_affine_coordinates' is deprecated [-Wdeprecated-declarations]
        int r = EC_KEY_set_public_key_affine_coordinates(ec_pubKey, bn_r, bn_s);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1122:1: note: 'EC_KEY_set_public_key_affine_coordinates' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 int EC_KEY_set_public_key_affine_coordinates(EC_KEY *key,
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cafe/IOSU/legacy/iosu_crypto.cpp:359:13: warning: 'd2i_RSAPrivateKey' is deprecated [-Wdeprecated-declarations]
                pkeyRSA = d2i_RSAPrivateKey(nullptr, (const unsigned char **)&pkeyTempPtr, pkeyData->size());
                          ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/rsa.h:309:41: note: 'd2i_RSAPrivateKey' has been explicitly marked deprecated here
DECLARE_ASN1_ENCODE_FUNCTIONS_name_attr(OSSL_DEPRECATEDIN_3_0,
                                        ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cafe/IOSU/legacy/iosu_crypto.cpp:367:3: warning: 'EVP_PKEY_assign' is deprecated [-Wdeprecated-declarations]
                EVP_PKEY_assign_RSA(evpPkey, pkeyRSA);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/evp.h:496:41: note: expanded from macro 'EVP_PKEY_assign_RSA'
#  define EVP_PKEY_assign_RSA(pkey,rsa) EVP_PKEY_assign((pkey),EVP_PKEY_RSA,\
                                        ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/evp.h:1327:1: note: 'EVP_PKEY_assign' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cafe/IOSU/legacy/iosu_crypto.cpp:399:8: warning: 'SSL_CTX_use_RSAPrivateKey' is deprecated [-Wdeprecated-declarations]
                        if (SSL_CTX_use_RSAPrivateKey(ctx, iosuCryptoCertificates.certList[i].pkey) != 1)
                            ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ssl.h:1809:1: note: 'SSL_CTX_use_RSAPrivateKey' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
5 warnings generated.
[322/403] Building CXX object src/Cemu/CMakeFiles/CemuComponents.dir/ncrypto/ncrypto.cpp.o
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:364:22: warning: 'ECDH_compute_key' is deprecated [-Wdeprecated-declarations]
                int sharedKeyLen = ECDH_compute_key(sharedKey, sizeof(sharedKey), ec_publicKey, ec_privKey, nullptr);
                                   ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1296:1: note: 'ECDH_compute_key' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 int ECDH_compute_key(void *out, size_t outlen,
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:366:3: warning: 'EC_KEY_free' is deprecated [-Wdeprecated-declarations]
                EC_KEY_free(ec_privKey);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1003:1: note: 'EC_KEY_free' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 void EC_KEY_free(EC_KEY *key);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:485:29: warning: 'EC_KEY_get0_private_key' is deprecated [-Wdeprecated-declarations]
                const BIGNUM* bnPrivKey = EC_KEY_get0_private_key(key);
                                          ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1048:1: note: 'EC_KEY_get0_private_key' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 const BIGNUM *EC_KEY_get0_private_key(const EC_KEY *key);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:494:24: warning: 'EC_KEY_new_by_curve_name' is deprecated [-Wdeprecated-declarations]
                EC_KEY* ec_privKey = EC_KEY_new_by_curve_name(NID_sect233r1);
                                     ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:998:1: note: 'EC_KEY_new_by_curve_name' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 EC_KEY *EC_KEY_new_by_curve_name(int nid);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:495:3: warning: 'EC_KEY_set_private_key' is deprecated [-Wdeprecated-declarations]
                EC_KEY_set_private_key(ec_privKey, bn_privKey);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1056:1: note: 'EC_KEY_set_private_key' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 int EC_KEY_set_private_key(EC_KEY *key, const BIGNUM *prv);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:516:23: warning: 'EC_KEY_new_by_curve_name' is deprecated [-Wdeprecated-declarations]
                EC_KEY* ec_pubKey = EC_KEY_new_by_curve_name(NID_sect233r1);
                                    ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:998:1: note: 'EC_KEY_new_by_curve_name' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 EC_KEY *EC_KEY_new_by_curve_name(int nid);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:517:11: warning: 'EC_KEY_set_public_key_affine_coordinates' is deprecated [-Wdeprecated-declarations]
                int r = EC_KEY_set_public_key_affine_coordinates(ec_pubKey, bn_x, bn_y);
                        ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1122:1: note: 'EC_KEY_set_public_key_affine_coordinates' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 int EC_KEY_set_public_key_affine_coordinates(EC_KEY *key,
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:624:11: warning: 'ECDSA_do_verify' is deprecated [-Wdeprecated-declarations]
                int r = ECDSA_do_verify(hash, sizeof(hash), ecSig, ecPubKey);
                        ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1391:1: note: 'ECDSA_do_verify' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 int ECDSA_do_verify(const unsigned char *dgst, int dgst_len,
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:627:3: warning: 'EC_KEY_free' is deprecated [-Wdeprecated-declarations]
                EC_KEY_free(ecPubKey);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1003:1: note: 'EC_KEY_free' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 void EC_KEY_free(EC_KEY *key);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:639:20: warning: 'ECDSA_do_sign' is deprecated [-Wdeprecated-declarations]
                ECDSA_SIG* sig = ECDSA_do_sign(hash, sizeof(hash), ec_privKey);
                                 ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1365:1: note: 'ECDSA_do_sign' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 ECDSA_SIG *ECDSA_do_sign(const unsigned char *dgst,
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:640:3: warning: 'EC_KEY_free' is deprecated [-Wdeprecated-declarations]
                EC_KEY_free(ec_privKey);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1003:1: note: 'EC_KEY_free' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 void EC_KEY_free(EC_KEY *key);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:689:24: warning: 'EC_KEY_new_by_curve_name' is deprecated [-Wdeprecated-declarations]
                EC_KEY* ec_keyPair = EC_KEY_new_by_curve_name(NID_sect233r1);
                                     ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:998:1: note: 'EC_KEY_new_by_curve_name' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 EC_KEY *EC_KEY_new_by_curve_name(int nid);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:690:3: warning: 'EC_KEY_generate_key' is deprecated [-Wdeprecated-declarations]
                EC_KEY_generate_key(ec_keyPair);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1101:1: note: 'EC_KEY_generate_key' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 int EC_KEY_generate_key(EC_KEY *key);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:695:3: warning: 'EC_KEY_free' is deprecated [-Wdeprecated-declarations]
                EC_KEY_free(ec_keyPair);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1003:1: note: 'EC_KEY_free' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 void EC_KEY_free(EC_KEY *key);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:705:20: warning: 'ECDSA_do_sign' is deprecated [-Wdeprecated-declarations]
                ECDSA_SIG* sig = ECDSA_do_sign(hash, hashLen, ec_privKey);
                                 ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1365:1: note: 'ECDSA_do_sign' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 ECDSA_SIG *ECDSA_do_sign(const unsigned char *dgst,
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:706:3: warning: 'EC_KEY_free' is deprecated [-Wdeprecated-declarations]
                EC_KEY_free(ec_privKey);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1003:1: note: 'EC_KEY_free' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 void EC_KEY_free(EC_KEY *key);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:710:18: warning: 'ECDSA_do_verify' is deprecated [-Wdeprecated-declarations]
                bool isValid = ECDSA_do_verify(hash, hashLen, sig, ec_pubKey) == 1;
                               ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1391:1: note: 'ECDSA_do_verify' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 int ECDSA_do_verify(const unsigned char *dgst, int dgst_len,
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:711:3: warning: 'EC_KEY_free' is deprecated [-Wdeprecated-declarations]
                EC_KEY_free(ec_pubKey);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1003:1: note: 'EC_KEY_free' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 void EC_KEY_free(EC_KEY *key);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:732:12: warning: 'ECDSA_do_verify' is deprecated [-Wdeprecated-declarations]
                bool r = ECDSA_do_verify(hash, hashLen, ecdsa_sig, ec_pubKey) == 1;
                         ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1391:1: note: 'ECDSA_do_verify' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 int ECDSA_do_verify(const unsigned char *dgst, int dgst_len,
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
/home/dougthor42/dev/Cemu/src/Cemu/ncrypto/ncrypto.cpp:734:3: warning: 'EC_KEY_free' is deprecated [-Wdeprecated-declarations]
                EC_KEY_free(ec_pubKey);
                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/ec.h:1003:1: note: 'EC_KEY_free' has been explicitly marked deprecated here
OSSL_DEPRECATEDIN_3_0 void EC_KEY_free(EC_KEY *key);
^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:182:49: note: expanded from macro 'OSSL_DEPRECATEDIN_3_0'
#   define OSSL_DEPRECATEDIN_3_0                OSSL_DEPRECATED(3.0)
                                                ^
/home/dougthor42/dev/Cemu/build/vcpkg_installed/x64-linux/share/rapidjson/../../include/openssl/macros.h:62:52: note: expanded from macro 'OSSL_DEPRECATED'
#     define OSSL_DEPRECATED(since) __attribute__((deprecated))
                                                   ^
20 warnings generated.
[365/403] Building CXX object src/audio/CMakeFiles/CemuAudio.dir/CubebInputAPI.cpp.o
/home/dougthor42/dev/Cemu/src/audio/CubebInputAPI.cpp:67:47: warning: bitwise operation between different enumeration types ('(anonymous enum at /home/dougthor42/dev/Cemu/dependencies/cubeb/include/cubeb/cubeb.h:190:1)' and 'cubeb_channel') is deprecated [-Wdeprecated-anon-enum-enum-conversion]
                input_params.layout = CUBEB_LAYOUT_QUAD_LFE | CHANNEL_FRONT_CENTER;
                                      ~~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~
1 warning generated.
[377/403] Building CXX object src/util/CMakeFiles/CemuUtil.dir/Zir/Core/IR.cpp.o
/home/dougthor42/dev/Cemu/src/util/Zir/Core/IR.cpp:210:79: warning: format specifies type 'unsigned long long' but the argument has type 'ZpIR::LocationSymbolName' (aka 'unsigned long') [-Wformat]
                        printf("   reg: %s sym:0x%llx\n", getRegisterName(block, itr.reg).c_str(), itr.name);
                                                 ~~~~                                              ^~~~~~~~
                                                 %lx
/home/dougthor42/dev/Cemu/src/util/Zir/Core/IR.cpp:214:79: warning: format specifies type 'unsigned long long' but the argument has type 'ZpIR::LocationSymbolName' (aka 'unsigned long') [-Wformat]
                        printf("   reg: %s sym:0x%llx\n", getRegisterName(block, itr.reg).c_str(), itr.name);
                                                 ~~~~                                              ^~~~~~~~
                                                 %lx
2 warnings generated.
[403/403] Linking CXX executable /home/dougthor42/dev/Cemu/bin/Cemu_release
/usr/bin/ld: warning: size of symbol `eglGetCurrentDisplay' changed from 22 in OpenGLRenderer.cpp.o (symbol from plugin) to 8 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `eglGetCurrentDisplay' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: size of symbol `eglSwapInterval' changed from 58 in OpenGLRenderer.cpp.o (symbol from plugin) to 8 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `eglSwapInterval' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glActiveShaderProgram' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glActiveTexture' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glAttachShader' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBeginQueryIndexed' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBeginTransformFeedback' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindAttribLocation' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindBuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindBufferBase' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindBufferRange' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindFragDataLocation' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindFramebuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindProgramPipeline' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindSampler' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindTexture' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindTextureUnit' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindVertexArray' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBindVertexBuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBlendColor' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBlendEquation' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBlendEquationSeparate' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBlendEquationSeparatei' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBlendFuncSeparate' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBlendFuncSeparatei' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBufferData' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBufferStorage' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glBufferSubData' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCheckFramebufferStatus' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClampColor' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClear' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClearColor' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClearDepth' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClearStencil' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClearTexImage' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClearTexSubImage' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClientActiveTexture' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClientWaitSync' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glClipControl' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glColorMaski' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCompileShader' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCompressedTexImage2D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCompressedTexImage3D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCompressedTexSubImage2D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCompressedTexSubImage3D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCompressedTextureSubImage2D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCompressedTextureSubImage3D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCopyBufferSubData' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCopyImageSubData' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCopyNamedBufferSubData' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCreateBuffers' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCreateFramebuffers' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCreateProgram' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCreateShader' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCreateShaderProgramv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCreateVertexArrays' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glCullFace' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDebugMessageCallback' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDebugMessageControl' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDeleteBuffers' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDeleteFramebuffers' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDeleteProgram' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDeleteProgramPipelines' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDeleteQueries' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDeleteShader' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDeleteSync' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDeleteTextures' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDeleteVertexArrays' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDepthFunc' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDepthMask' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDepthRange' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDepthRangedNV' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDetachShader' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDisable' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDisableVertexArrayAttrib' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDisableVertexAttribArray' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDisablei' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDrawArrays' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDrawArraysInstanced' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDrawBuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDrawBuffers' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDrawElementsBaseVertex' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDrawElementsInstancedBaseVertexBaseInstance' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDrawRangeElements' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glDrawRangeElementsBaseVertex' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glEnable' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glEnableVertexArrayAttrib' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glEnableVertexAttribArray' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glEnablei' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glEndQueryIndexed' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glEndTransformFeedback' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glFenceSync' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glFinish' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glFlush' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glFlushMappedBufferRange' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glFramebufferTexture2D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glFramebufferTextureLayer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glFrontFace' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGenBuffers' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGenFramebuffers' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGenProgramPipelines' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGenQueries' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGenSamplers' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGenTextures' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGenVertexArrays' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetAttachedShaders' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetAttribLocation' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetBufferSubData' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetError' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetIntegerv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetNamedBufferSubData' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetProgramBinary' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetProgramInfoLog' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetProgramResourceIndex' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetProgramiv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetQueryIndexediv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetQueryObjecti64v' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetQueryObjectiv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetShaderInfoLog' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetShaderiv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetString' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetTexImage' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetTexLevelParameteriv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetTextureLevelParameteriv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetTextureSubImage' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetUniformBlockIndex' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glGetUniformLocation' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glInvalidateFramebuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glInvalidateNamedFramebufferData' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glInvalidateTexImage' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glIsEnabled' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glLinkProgram' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glLogicOp' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glMapBuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glMapBufferRange' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glMapNamedBuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glMapNamedBufferRange' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glMaxShaderCompilerThreadsARB' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glMemoryBarrier' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glNamedBufferStorage' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glNamedBufferSubData' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glNamedFramebufferTexture' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glNamedFramebufferTexture2DEXT' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glNamedFramebufferTextureLayer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glObjectLabel' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glPixelStorei' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glPointSize' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glPolygonMode' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glPolygonOffset' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glPolygonOffsetClampEXT' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glPrimitiveRestartIndex' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glProgramBinary' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glProgramParameteri' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glProgramUniform1fv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glProgramUniform1i' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glProgramUniform1iv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glProgramUniform2fv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glProgramUniform2i' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glProgramUniform4iv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glQueryCounter' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glSamplerParameterf' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glSamplerParameterfv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glSamplerParameteri' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glSamplerParameteriv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glScissor' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glShaderSource' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glShaderStorageBlockBinding' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glStencilFuncSeparate' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glStencilMaskSeparate' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glStencilOpSeparate' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexImage1D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexImage2D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexImage3D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexParameterf' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexParameterfv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexParameteri' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexStorage1D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexStorage2D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexStorage3D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexSubImage1D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexSubImage2D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTexSubImage3D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTextureBarrier' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTextureParameteri' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTextureSubImage2D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTextureSubImage3D' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTextureView' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glTransformFeedbackVaryings' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glUniform1i' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glUniform2f' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glUniform2i' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glUniform4fv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glUniformBlockBinding' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glUniformMatrix4fv' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glUnmapBuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glUseProgram' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glUseProgramStages' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexArrayAttribBinding' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexArrayAttribIFormat' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexArrayBindingDivisor' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexArrayElementBuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexArrayVertexBuffer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexAttribBinding' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexAttribDivisor' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexAttribFormat' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexAttribIFormat' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexAttribIPointer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexAttribPointer' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glVertexBindingDivisor' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glViewport' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
/usr/bin/ld: warning: type of symbol `glViewportIndexedf' changed from 2 to 1 in /tmp/lto-llvm-b18d3d.o
```

</details>

--- a/BUILD.md
+++ b/BUILD.md
@@ -63,7 +63,7 @@ You can use GCC by doing the following:
  - If step 4 gives you an error that contains something like `main.cpp.o: in function 'std::__cxx11::basic_string...`, you likely are experiencing a clang-14 issue. This can only be fixed by either lowering the clang version or using GCC, see below.
  - If step 4 gives you a different error, you could report it to this repo or try using GCC. Just make sure your standard library and compilers are updated since Cemu uses a lot of modern features!
  - If step 4 gives you undefined libdecor_xx, you are likely experiencing an issue with sdl2 package that comes with vcpkg. Delete sdl2 from vcpkg.json in source file and recompile.
- - If step 4 gives you `fatal error: 'span' file not found`, then you're missing `libstdc++`. Install with your package manager, eg `sudo apt install libstdc++-10-dev`. See #644.
+ - If step 4 gives you `fatal error: 'span' file not found`, then you're either missing `libstdc++` or are using a version that's too old. Install at least v10 with your package manager, eg `sudo apt install libstdc++-10-dev`. See #644.
 
 ## macOS
 

--- a/BUILD.md
+++ b/BUILD.md
@@ -59,10 +59,11 @@ You can use GCC by doing the following:
  - If step 3 fails while compiling the boost-build dependency, it means you don't have a working/good standard library installation. Check the integrity of your system headers and making sure that C++ related packages are installed and intact.
  - If step 3 gives a random error, read the `[package-name-and-platform]-out.log` and `[package-name-and-platform]-err.log` for the actual reason to see if you might be lacking the headers from a dependency.
  - If step 3 is still failing or if you're not able to find the cause, please make an issue on our Github about it!
+ - If step 3 fails during rebuild after `git pull` with an error that mentions RPATH, add this to the end of step 3: `-DCMAKE_BUILD_WITH_INSTALL_RPATH=ON`
  - If step 4 gives you an error that contains something like `main.cpp.o: in function 'std::__cxx11::basic_string...`, you likely are experiencing a clang-14 issue. This can only be fixed by either lowering the clang version or using GCC, see below.
  - If step 4 gives you a different error, you could report it to this repo or try using GCC. Just make sure your standard library and compilers are updated since Cemu uses a lot of modern features!
  - If step 4 gives you undefined libdecor_xx, you are likely experiencing an issue with sdl2 package that comes with vcpkg. Delete sdl2 from vcpkg.json in source file and recompile.
- - If step 3 fails during rebuild after `git pull` with an error that mentions RPATH, add this to the end of step 3: `-DCMAKE_BUILD_WITH_INSTALL_RPATH=ON`
+ - If step 4 gives you `fatal error: 'span' file not found`, then you're missing `libstdc++`. Install with your package manager, eg `sudo apt install libstdc++-10-dev`. See #644.
 
 ## macOS
 

